### PR TITLE
[TASK]	Update readme installation description

### PR DESCRIPTION
**Pull request by riaspirjak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,25 @@ With this plugin, you can merge duplicate property groups in Shopware 6.
 ## Blog post
 Please find more details in our [blog post](https://webwirkung.ch/blog/duplikate-von-eigenschaften-in-shopware-6-zusammenfuehren/).
 
-## Installation
-Download the ZIP file of this repository and install it via the Shopware 6 administration or use composer.
+## Installalation
```
Comment (noahmenzi) [inline README.md:7]: Typo in the title.

File: README.md [hunk 2]
```diff
@@ -4,8 +4,25 @@ With this plugin, you can merge duplicate property groups in Shopware 6.
...
+Add access rights for this repository to the github token of your client.
```
Comment (noahmenzi) [inline README.md:8]: Please notice that this is a public Github repository. I would leave the part about downloading a ZIP and uploading it in the backend here and add the installation with composer as a second option. Thanks!

**Review by noahmenzi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (riaspirjak) [reply README.md]: @noahmenzi thanks I fixed it ;)

File: README.md (hunk 2 above)
Comment (riaspirjak) [reply README.md]: @noahmenzi added!

**Review by noahmenzi — APPROVED** [review]

**Pull request merged**